docs/advanced_usage/8_logging.md

```diff
@@ -96,24 +96,23 @@ The runhistory.json in split into four parts. `stats`, `data`, `configs`, and `c
   },
 ```
 
-`data` contains a list of entries, one for each configuration.
+`data` contains a list of entries, one for each configuration where the keys are the one-based `config_id`.
 ```json
-  "data": [
-    [
-      1,                            # config_id
-      null,                         # instance or None
-      209652396,                    # seed or None
-      null,                         # budget or None
-      5.4345623938566385,           # cost
-      6.699562072753906e-05,        # time
-      6.299999999992423e-05,        # cpu_time
-      1,                            # status
-      1733133181.2144582,           # start_time
-      1733133181.21695,             # end_time
-      {}                            # additional_info
-    ],
+  "data": {
```

Comment:
AFAIK the config id is a unique configuration *without* the budget and the instance. If this is the case, we need a different data structure, because currently we would overwrite evaluated trials with the same configs but different budgets/instances. I would propose a list of dictionaries.

Reply:
I will adapt this accordingly